ng-ncc-app/src/app/components/agent/agent.component.html

```diff
@@ -0,0 +1,7 @@
+<!-- Agent's avatar. -->
+<img class="agent__avatar" src="assets/img/avatar.svg" alt="">
```

Comment:
can we fill the alt attribute?

Reply:
What would the alt attribute say?

Reply:
Generally `alt` should describe what's the image/log is in a way that makes it clear to understand for people navigating the website using a screen reader. If it's just a decorative element and doesn't add any value then it should be hidden with an ARIA rule.